src/background.js

```diff
@@ -44,6 +45,13 @@ async function openOverview() {
     }
 }
 
+const openOverviewURL = url => chrome.tabs.create({ url })
```

Comment:
Also, with the `openOverviewURL` function, why does it differ from `openOptionsURL` function in taking an absolute URL rather than just the path? I think you could simplify it a lot by just hardcoding the overview URL in there and having a `query` arg to append to it. Then content script side just needs to send the query; no need to know what the URL is.

Reply:
Yeah right. I didn't know why I did that .-.
I will update it!